src/main/java/frc/robot/Constants.java

```diff
@@ -115,41 +115,44 @@ public static class PPConstants {
             .withSteerMotorInverted(STEER_MOTOR_REVERSED);
 
     // CHANGE FOR NEW ROBOT:
-    // Front Right
-    private static final int FRONT_LEFT_DRIVE_MOTOR_ID = 0;
-    private static final int FRONT_LEFT_STEER_MOTOR_ID = 7;
-    private static final int FRONT_LEFT_ENCODER_ID = 11;
-    private static final double FRONT_LEFT_ENCODER_OFFSET = -0.091796875;
-
-    private static final double FRONT_LEFT_X_POS_INCHES = 12.25;
-    private static final double FRONT_LEFT_Y_POS_INCHES = 12.25;
+    private static final double moveCOMY = 0.046007;
+    private static final double moveCOMX = -3.36044;
```

Comment:
this should be positive, the COM is closer to the front of the robot than the back